packages/react-server-cli/src/callerDependency.js

```diff
@@ -1,7 +1,17 @@
 import lookup from "look-up";
+import path from "path";
 
 export default function callerDependency(dep) {
 	// TODO: We should grab stuff based on what the routes file would get out
 	// of `require.resolve(dep)`.  Using `process.cwd()` instead for now.
-	return lookup("node_modules/" + dep, {cwd: process.cwd()});
+	let cwd = process.cwd(),
+		lookupResult;
+
+	if (process.env.NODE_ENV === '__react-server-cli-unit-test__') { // eslint-disable-line no-process-env
```

Comment:
I'm not sure if it's better in the overall scheme of things (mockery is black magic), but could we use `mockery` to intercept requires of `callerDependency` from the spec itself? At least that way the complexity is in the spec, rather than here.